stream_alert/rule_processor/classifier.py

```diff
@@ -308,7 +308,13 @@ def _convert_type(self, parsed_data, schema, options):
                     return False
 
             elif value == 'boolean':
-                payload[key] = payload[key].lower() == 'true'
+                if type(payload[key]) in (str, unicode):
```

Comment:
this seems overly complex - can we just cast it to a string instead? if it's already a string it doesn't matter, and if it's a boolean it'll cast to `True` or `False`:
`payload[key] = str(payload[key]).lower() == 'true'`